src/player/player.js

```diff
@@ -0,0 +1,23 @@
+export default function Player(name, board) {
+	this.name = name;
+	this.board  = board;
+	this.playAIPos = [];		
+    
+	this.randomPlay = function () {
+	let line, col;	
+	let cond = true;
+      if (this.name === 'Computer' && this.playAIPos.length < 100){
+        while (cond){
+          line = Math.floor(Math.random() * Math.floor(9));
+          col  = Math.floor(Math.random() * Math.floor(9));
+          if (this.playAIPos.indexOf([line, col]) === -1){
+            this.playAIPos.push([line, col]);
+            cond = false;
+            console.log('AIpos:', this.playAIPos);
```

Comment:
Remove `console.log` from code, it affects the performance of code and it's ugly.